pkg/kubefed2/disable.go

```diff
@@ -202,8 +301,13 @@ func deleteFederatedType(config *rest.Config, typeConfig typeconfig.Interface, w
 	crdName := typeconfig.GroupQualifiedName(typeConfig.GetFederatedType())
 	err = client.CustomResourceDefinitions().Delete(crdName, nil)
```

Comment:
The latter is not implemented in this PR due to the following per our offline conversation:

1. Needing to discover the federation config (it has since been merged).
2. Possibly resolving it with docs instead of code due to member cluster resources.

With that in mind, are you suggesting we implement this in code before merging this or that we should at least add a `--force` option prior to merging?